Tasks/Kasinskiy/Task2/task2.py

```diff
@@ -0,0 +1,54 @@
+import datetime
+
+dict_hours = {1:('один час', 'второго', 'два'), 2:('два часа', 'третьего', 'три'), 3:('три часа', 'четвертого', 'четыре'), 4:('четыре часа', 'пятого', 'пять'), 5:('пять часов', 'шестого', 'шесть'), 6:('шесть часов', 'седьмого', 'семь'),
+     7:('семь часов', 'восьмого', 'восемь'), 8:('восемь часов', 'девятого', 'девять'), 9:('девять часов', 'десятого', 'десять'), 10:('десять часов', 'одиннадцатого', 'одиннадцать'), 11:('одиннадцать часов', 'двенадцатого', 'двенадцать'),
+     0:('двенадцать часов', 'первого', 'час')}
+
+dict_minutes = {1:'одна минута', 2:'две минуты', 3:'три минуты', 4:'четыре минуты', 5:'пять минут', 6:'шесть минут', 7:'семь минут', 8:'восемь минут', 9:'девять минут', 10:'десять минут', 11:'одиннадцать минут', 12:'двенадцать минут',
+      13: 'тринадцать минут', 14: 'четырнадцать минут', 15: 'пятнадцать минут', 16: 'шестнадцать минут', 17: 'семнадцать минут', 18: 'восемнадцать минут', 19: 'девятнадцать минут', 20: ('двадцать минут', 'двадцать'),
+      30: ('половина', 'тридцать'),  40: ('сорок минут', 'сорок'), 45:'без пятнадцати минут', 46:'без четырнадцати минут', 47:'без тринадцати минут', 48:'без двенадцати минут', 49:'без одиннадцати минут', 50:'без десяти минут',
+      51:'без девяти минут', 52:'без восьми минут', 53:'без семи минут', 54:'без шести минут', 55:'без пяти минут', 56:'без четырех минут', 57:'без трех минут', 58:'без двух минут', 59:'без одной минуты'}
+
+time_entry = input('Enter your time in the format hh:mm, or enter * to get the current time: ')
```

Comment:
if you propose some format for a user you should at least check some basic parameters of that input like length and ':' sign